.bcr/presubmit.yml

```diff
@@ -1,7 +1,7 @@
 bcr_test_module:
   module_path: "e2e"
   matrix:
-    bazel: ["8.x", "7.x"]
+    bazel: ["8.x"]
```

Comment:
is it broken on bazel 7 by this change?

Reply:
no it was broken before. i am just here to have a green entry on BCR

Reply:
CI is only testing with 8.x as well.